tests/run.go

```diff
@@ -311,14 +313,6 @@ func main() {
 	}
 }
 
-func toolPath(name string) string {
```

Comment:
My understanding is that this function was not called from anywhere. Is that correct?

Reply:
Yep, it was unused, so I deleted it.

It's unused in the [upstream `test/run.go`](https://github.com/golang/go/blob/cfe2ab42e764d2eea3a3339aac1eaff97520baa0/test/run.go) file too. I plan to send a CL to remove it from there too after the tree opens up.